lib/slack-ruby-bot-server/ext.rb

```diff
@@ -0,0 +1,3 @@
+%w(bson/object_id grape/sort_extension slack-ruby-bot).each do |ext|
```

Comment:
I feel like it's almost easier to write the 3 requires in a row explicitly and call it a day.